tdp/cli/commands/conftest.py

```diff
@@ -1,11 +1,43 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+from collections.abc import Generator
 from pathlib import Path
+from typing import NamedTuple
 
 import pytest
+from click.testing import CliRunner
+from sqlalchemy import create_engine
 
+from tdp.cli.commands.init import init
 from tdp.conftest import generate_collection_at_path
+from tdp.core.models import BaseModel
+
+
+class TDPInitArgs(NamedTuple):
+    collection_path: Path
+    db_dsn: str
+    vars: Path
+
+
+@pytest.fixture
+def tdp_init(
+    collection_path: Path, db_dsn: str, vars: Path
+) -> Generator[TDPInitArgs, None, None]:
+    base_args = [
+        "--collection-path",
+        collection_path,
```

Comment:
`collection_path` is a `Path`, we should not have a list with multiple type for element, use `str(collection_path)` to convert `Path` to `str`. Same for all occurrences.

Reply:
done